dd-sdk-android-gradle-plugin/src/test/kotlin/com/datadog/gradle/plugin/DdAndroidGradlePluginTest.kt

```diff
@@ -145,10 +144,9 @@ internal class DdAndroidGradlePluginTest {
         assertThat(task.variantName).isEqualTo(flavorName)
         assertThat(task.versionName).isEqualTo(fakeExtension.versionName)
         assertThat(task.serviceName).isEqualTo(fakeExtension.serviceName)
-        assertThat(task.remoteRepositoryUrl).isEmpty()
         assertThat(task.site).isEqualTo(fakeExtension.site)
-        assertThat(task.mappingFilePath)
-            .isEqualTo("${fakeProject.buildDir}/outputs/mapping/$variantName/mapping.txt")
+        assertThat(task.remoteRepositoryUrl).isEqualTo(task.remoteRepositoryUrl)
+        assertThat(task.mappingFilePath).isEqualTo(fakeExtension.mappingFilePath)
```

Comment:
You need to build the `expectedPath` to handle both cases when the mapping file uses the default or when it's set by the customer.

Reply:
I have 2 tests for that: 
`𝕄 configure the upload task with the extension info 𝕎 configureVariant()` and `𝕄 use sensible defaults 𝕎 configureVariant() { empty config }`. The only problem I see here is that they are testing all the properties, if we chose to have dedicated tests for the `mappingFilePath` then I say we should have dedicated tests for each property in the `DdExtension` class.